### PR TITLE
feat(action-aws): lambda-deploy can update function alias

### PR DESCRIPTION
**Pull request by JoffreyPlouvier**
be able to update function alias after lambda code update

### CODE REVIEW

**Review by fmiquel90 — APPROVED** [review]

File: actions/aws/lambda-deploy/action.yml [hunk 1]
```diff
@@ -42,6 +46,7 @@ runs:
         aws-region: ${{ inputs.AWS_REGION }}
 
     - name: Update the Lambda code (new Docker image)
+      id: update_function
```
Comment (ikodev) [inline actions/aws/lambda-deploy/action.yml:49]: Two githubActions can have the same ID ? Here `update_function` for both "update the lambda code" and "update the lambda alias".

Comment (ikodev) [reply actions/aws/lambda-deploy/action.yml:49]: https://docs.github.com/fr/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsid

Comment (ikodev) [reply actions/aws/lambda-deploy/action.yml:49]: `update_function_name` ?

File: actions/aws/lambda-deploy/action.yml [hunk 2]
```diff
@@ -51,4 +56,17 @@ runs:
         aws lambda wait function-updated --function-name $LAMBDA_FUNCTION_NAME
         result=$(aws lambda publish-version --function-name $LAMBDA_FUNCTION_NAME)
         version=$(echo ${result}| jq -r .Version)
-        echo "Deployed version ${version} of the lambda $LAMBDA_FUNCTION_NAME"
\ No newline at end of file
+        echo "lambda_version=$version" >> $GITHUB_OUTPUT
+        echo "Deployed version ${version} of the lambda $LAMBDA_FUNCTION_NAME"
+
+    - name: Update the Lambda Alias
+      id: update_function
```
Comment (ikodev) [inline actions/aws/lambda-deploy/action.yml:63]: `update_function_alias` ?

File: actions/aws/lambda-deploy/action.yml (hunk 1 above)
Comment (JoffreyPlouvier) [reply actions/aws/lambda-deploy/action.yml:49]: nice catch

**Pull request merged**